sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -158,3 +207,16 @@ function createParseError(response: PipelineResponse, err: any): RestError {
     response: response,
   });
 }
+
+function isBinaryContentType(contentType: string) {
+  return [
+    "application/octet-stream",
+    "application/x-rdp",
+    "image/bmp",
+    "image/gif",
+    "image/jpeg",
+    "image/png",
+    "application/pdf",
+    "application/zip",
```

Comment:
Does this containes all possible binary content-type ?

Reply:
No, this is only a small set of "common" content-types. We don't want to include a big set as that would impact bundle size. As an alternative, we allow setting a `binaryResponse`  option when sending the request to explicitly say that you are expecting a binary.

Reply:
I see. Thanks for the explaination